src/main/scala/no/ndla/learningpathapi/controller/InternController.scala

```diff
@@ -105,13 +107,18 @@ trait InternController {
       }
     }
 
-    get("/dump/learningpath") {
+    get("/dump/learningpath/?") {
```

Comment:
Burde du ikkje hatt en metode for å hente ut en læringssti fra apiet? Det er jo ikkje publiserte stier som skal migreres, men stier oppretta av eksterne og nesten ingen av disse er publiserte.

Reply:
Ah riktig, jeg legger til et flag (default off for å ikke ødelegge for search-api) for å få med alt

Reply:
Men vi skal ikkje migrere alle, kun dei vi har id på, på samme måte som for alt anna,

Reply:
Yap, det skjønner jeg, men når vi henter ut læringsstier så henter vi ut alle sidene og filtrerer vekk de som ikke er med i lista. Enkelid endepunktet brukte bare når du ga en og en id med `--id` og ikke med `--id-file`.

Reply:
Jeg kan legge på enkelid endepunktet for consistency om vi vil, men jeg tror ikke det kommer til å bli brukt

Reply:
Det er ikkje så farlig. Eg trudde det var det endepunktet som blei brukt.